common/src/main/java/org/dash/wallet/common/ui/FancyAlertDialog.kt

```diff
@@ -1,203 +0,0 @@
-/*
- * Copyright 2020 Dash Core Group
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.dash.wallet.common.ui
-
-import android.graphics.Color
-import android.graphics.drawable.ColorDrawable
-import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import android.view.ViewGroup.LayoutParams
-import android.view.Window
-import android.widget.ImageView
-import android.widget.TextView
-import androidx.annotation.DrawableRes
-import androidx.annotation.StringRes
-import androidx.fragment.app.DialogFragment
-import androidx.fragment.app.FragmentManager
-import org.dash.wallet.common.R
-import org.dash.wallet.common.UserInteractionAwareCallback
-import org.dash.wallet.common.databinding.FancyAlertDialogBinding
-
-@Deprecated("Use AdaptiveDialog")
-class FancyAlertDialog : DialogFragment() {
-    var onFancyAlertButtonsClickListener: FancyAlertButtonsClickListener? = null
-    private val binding by viewBinding(FancyAlertDialogBinding::bind)
```

Comment:
This will cause merge problems in DashPay. However, I think the AdaptiveDialog handles all of the use cases in DashPay with respect to FancyAlertDialogs, including the progress bar dialogs.

Reply:
Right. During the last merge, I moved some of the `FancyAlertDialog` usages to the `AdaptiveDialog`, but there are still some left. We can handle them next merge which could take some work in general, given all the changes with payment flows.